code/game/gamemodes/miniantags/borer/borer_event.dm

```diff
@@ -13,16 +13,14 @@
 /datum/event/borer_infestation/announce()
 	if(successSpawn)
 		GLOB.command_announcement.Announce("Unidentified lifesigns detected coming aboard [station_name()]. Secure any exterior access, including ducting and ventilation.", "Lifesign Alert", new_sound = 'sound/AI/aliens.ogg')
+	else
+		var/text = "Warning: Could not spawn any mobs for event Borer Infestation"
+		message_admins(text)
+		log_adminwarn(text)
```

Comment:
Why define text here? Also `log_and_message_admins()` is a proc that exists which may be useful for this

Reply:
Oh, I was aware of that proc but I saw `usr` so I thought it wouldn't really work but after testing, it does work. I'll fix that and the others